docs/sdk/features/edgedb.md

```diff
@@ -0,0 +1,60 @@
+# EdgeDB Feature
+
+## Overview
+
+The EdgeDB feature allows the saving of user data into our EdgeDB storage and to be able to use the data in segmenting decisions without needing
+to pass that data in our SDKs every time. 
+
+## Usage
+
+### Rest API
+
+We are able to support updates to users using our EdgeDB Public Rest API. The docs for it can be found here. (Link to the completed API docs)
+
+You are able to use this update users with all of the supported data, along with custom data, and be able to use that data for segmenting in the
+SDKs without having to explicitly pass all of the data when `identifyUser` is called.
+
+```
+const user = {
+    user_id: 'my-user',
+    customData: {
+        amountSpent: 50
+    }
+}
+
+await axiosClient.request({
```

Comment:
given that this is an example of an API call, should it be a curl request rather than Javascript?